<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewUser</class>
 <widget class="QDialog" name="AddNewUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>153</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>ОК</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>12</y>
     <width>178</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Добавить нового пользователя</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>220</width>
     <height>60</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QSplitter" name="splitter_3">
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Логин</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit"/>
    </widget>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Пароль</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2"/>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_4">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Секретное слово</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>AddNewUser</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
